pylintrc

```diff
@@ -17,7 +17,7 @@
 [MASTER]
 
 # The number of jobs to spawn for linting.
-jobs=8
+jobs=2
```

Comment:
Why this change?  How is 2 better?  Seems we want more jobs to be done faster?

Reply:
The timing was no different. I reverted this as it shouldn't be in this PR.